packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -106,8 +105,13 @@ export class CameraTargetInput extends ConnectedLitElement {
   @property({attribute: false}) change?: (newValue: Vector3D) => void;
   @internalProperty() target?: Vector3D;
 
+  // @ts-ignore
   stateChanged(state: State) {
-    this.target = getCamera(state).target ?? getInitialCamera(state).target;
+    if (getModelViewer() !== undefined && getModelViewer() !== null) {
+      this.target = getCameraState(getModelViewer()!).target;
```

Comment:
now using the target of model-viewer

Reply:
Also, if you put `const modelViewer = getModelViewer();` above the if statement, then you won't need the `!` either, because TS can figure out you already checked that it exists.